README.md

```diff
@@ -6,27 +6,42 @@ CKB Academy is committed to ensuring that new CKB developers have an easy learni
 Our mission is to foster a robust ecosystem around Nervos Network by providing developers with all the tools and resources they need to create innovative applications and solutions.
 We look forward to having you join us as part of our growing community.
 
-This project is going to build a comprehensive set of guides, tutorials, labs, and sandbox environments to help developers learn and test their knowledge of Nervos CKB, allowing developers to learn at their own pace.
+At the current stage, CKB Academy is dedicated to solving the problem of "scattered" or "fragmented" information, becoming a landing page for Nervos development resources, collecting and organizing various tutorials, how-to guides, references, explanations and other educational content worth highlighting.
+
+In the future, we hope CKB Academy will become a comprehensive [Documentation System][1] / Library / Playground etc.
+
+## Sitemap
+- Courses: a list of interactive courses covering various topics related to Nervos development
+- Library: a collection of how-to articles, references, explanations and other documentation resources
+- Playground (Labs): a set of sandbox environments where developers can experiment with Nervos development and test their code
+
 
 ## TODO list
 
-- [ ] Gather and organize more available study materials, transform them into interactive learning courses
-  - [x] [Refactor zero2ckb training course](https://github.com/Flouse/ckb-academy/issues/3) | send a pkp2h transaction
+- Gather and organize available study materials worth highlighting
   - [x] [Nervos Developer Training Course](https://nervos.gitbook.io/developer-training-course/)
+  - [ ] [Nervos Network Developers Resources Hub](https://talk.nervos.org/t/nervos-network-developers-resources-hub/7261)
+  
+- Interactive learning courses
+  - [x] [Refactor zero2ckb training course](https://github.com/Flouse/ckb-academy/issues/3) | send a pkp2h transaction
+     - https://academy.ckb.dev/courses/basic-theory
+     - https://academy.ckb.dev/courses/basic-operation
+  - [ ] [Write an SUDT Script by Capsule](https://docs.nervos.org/docs/labs/sudtbycapsule/)
   - [ ] Send a multisig transaction
   - [ ] Deploy a simple contract
   - [ ] Deploy a upgradable contract
   - [ ] etc.
+
+- Playground (Labs)
+  - [ ] [Time to Hello World on CKB](https://github.com/Flouse/ckb-tthw/tree/main/js)
+    -> demo: https://ckb-tthw.vercel.app
```

Comment:
Maybe [demo](https://ckb-tthw.vercel.app) is better here? There are too many styles, which may cause some confusion, and some users with small screens may see line breaks

Reply:
Oh sorry, I miss this `[Time to Hello World on CKB](https://github.com/Flouse/ckb-tthw/tree/main/js)`,  -> demo: https://ckb-tthw.vercel.app, is ok here